foodwise/src/main/java/br/com/foodWise/foodWise/service/UserService.java

```diff
@@ -1,4 +1,28 @@
 package br.com.foodWise.foodWise.service;
 
+import br.com.foodWise.foodWise.model.entities.User;
+import br.com.foodWise.foodWise.model.repositories.UserRepository;
+import br.com.foodWise.foodWise.service.exceptions.user.UserNotFound;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
 public class UserService {
+
+    private static final Logger logger = LoggerFactory.getLogger(UserService.class);
+
+    @Autowired
+    private UserRepository userRepository;
+
+
+    public Optional<User> getUser(Long userId) {
+
+        logger.info("Buscando usuario! ");
```

Comment:
Logs menores podemos usar como debug. https://stackoverflow.com/questions/6860186/log-info-vs-log-debug 

DEBUG: Information **interesting for Developers,** when trying to debug a problem.
INFO: Information interesting for Support staff trying to figure out the context of **a given error**
WARN to FATAL: Problems and Errors depending on the level of damage.